core/server/data/meta/index.js

```diff
@@ -71,35 +71,36 @@ function getMetaData(data, root) {
             logo: settingsCache.get('logo'),
             amp: settingsCache.get('amp')
         }
-    };
+    },
+        customExcerpt,
+        metaDescription,
+        fallbackExcerpt;
 
-    return Promise.props(getBlogLogo()).then(function (result) {
-        metaData.blog.logo = result;
+    metaData.blog.logo = getBlogLogo();
```

Comment:
For readability purpose: you could remove this single line and add the property to the `metaData` object directly, because it's not longer async.